controllers/kyma_controller_helper_test.go

```diff
@@ -237,6 +239,23 @@ func ModuleTemplatesExist(clnt client.Client, kyma *v1beta1.Kyma, remote bool) f
 	}
 }
 
+func WatcherLabelsAnnotationsExist(clnt client.Client, kyma *v1beta1.Kyma) func() error {
```

Comment:
Let's use the normal way to write the test function, not continue to introduce this `return func() error`, basically you can write this function to normal one to `return error`, like `func WatcherLabelsAnnotationsExist(clnt client.Client, kyma *v1beta1.Kyma) error`, so that it's more reusable. 

then in the Eventually condition, write like this:
```
Eventually(WatcherLabelsAnnotationsExist, Timeout, Interval).WithArguments(runtimeClient, kyma).Should(Succeed())
```
There are many existing test are refactored using this way, you can find some examples, like `updateModuleTemplateSpec`

Reply:
Oke, I can do that. I know both ways. I just used this way since the other test above was written in the same way

Reply:
this test is failing